Katello/katello
# Fixes #37983 - Remove syspurpose addons from Katello

#### What are the changes introduced in this pull request?
* There are no longer any SKUs that use syspurpose addons. Additionally, the addons command is being removed from subscription-manager in RHEL10. So we are free to remove addons from Katello
* Removed systempurpose_addons from AngularJS, React, and the Rails backend
* Created a DB Migration to remove the addon tables

#### What are the testing steps for this pull request?
* Checkout PR and apply migration, make sure tables are removed
* Look around in the UI to make sure nothing broke
* Look around in the code to see if I missed anything
* Make sure the system porpoise dolphin did not sneak in 🥚
